R/cleaning_drug_exposure_extract_from_bigQueryDB.R

```diff
@@ -0,0 +1,32 @@
+# Load necessary library
+library(readr)

```

Comment:
A comment will be added to refer to another script loaded for accessing BigQuery DB.

```suggestion
# Please refer to '[Accessing_BigQuery_DB.R](https://github.com/SAFEHR-data/ramses-package/blob/Zsenousy/32-access-and-clean-data-tables-extracted-from-BigQuery-DB/R/Accessing_BigQuery_DB.R) to learn how to generate extract of drug_exposure and concept tables. The resulted data tables will be uncleaned and will require further cleaning as shown here in this script
# Load necessary library
library(readr)

```